src/Compilers/CSharp/Portable/Symbols/RefKindExtensions.cs

```diff
@@ -30,5 +30,20 @@ public static RefKind GetRefKind(this SyntaxKind syntaxKind)
                     throw ExceptionUtilities.UnexpectedValue(syntaxKind);
             }
         }
+
+        public static bool IsWritableReference(this RefKind refKind)
+        {
+            switch (refKind)
+            {
+                case RefKind.Ref:
+                case RefKind.Out:
+                    return true;
+                case RefKind.None:
+                case RefKind.In:
+                    return false;
```

Comment:
Why aren't we checking RefReadOnly here?

Reply:
It is the same value as `RefKind.In`.